### PR TITLE
DD-272 Updating dependencies

### PR DESCRIPTION
**Pull request by imran-gov**
Allowing only Node 20 or 22.

Updated the following dependencies:
Adding Husky ^9.1.6 then adding ./husky/pre-push
express ^4.19.2 -> ^4.21.1
sinon ^18.0.0 -> ^19.0.2
eslint ^9.9.0 -> ^9.12.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,12 +29,12 @@
   },
   "homepage": "https://github.com/HMPO/hmpo-reqres",
   "dependencies": {
-    "express": "^4.19.2",
-    "sinon": "^18.0.0"
+    "express": "^4.21.0",
+    "sinon": "^19.0.2"
   },
   "devDependencies": {
```
Comment (SamChatfield) [inline package.json:36]: Need `husky` added and set up with pre-push hook

**Review by SamChatfield — CHANGES_REQUESTED** [review]
Let's also update the package `engines` to match the other libs

File: package.json (hunk 1 above)
Comment (SamChatfield) [reply package.json:36]: We're still missing the husky `prepare` package script meaning `npx husky init` hasn't been run

Comment (imran-gov) [reply package.json:36]: thanks Sam, updated now

**Review by cmcgee480 — APPROVED** [review]

**Review by SamChatfield — APPROVED** [review]
LGTM

**Pull request merged**